CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is up with this line?

Reply:
It's the starting line for an empty file.